_pages/en_US/homebrew-launcher-(pichaxx).txt

```diff
@@ -8,7 +8,7 @@ title: "Homebrew Launcher (PicHaxx)"
 
 This method of using Seedminer for further exploitation uses your `movable.sed` file to gain access to the Homebrew Launcher using the PicHaxx exploit for the purposes of injecting an exploitable DSiWare title into the DS Download Play application. This method requires you to already own (or download) the free "Pokemon Picross" game from the eShop.
 
-If your device already has access to the Homebrew Launcher (whether through pichaxx or another exploit), you can skip to [DSiWare Dumper](dsidumper)).
+If your device already has access to the Homebrew Launcher (whether through pichaxx or another exploit), you can skip to [Frogtool](installing-boot9strap-(frogtool)).
```

Comment:
Capitalize PicHaxx / follow whatever capitalization is used in the rest of the guide